packages/cli/src/controller/codegen-controller.ts

```diff
@@ -186,7 +187,10 @@ export async function codegen(projectPath: string): Promise<void> {
   const interfacesPath = path.join(projectPath, TYPE_ROOT_DIR, `interfaces.ts`);
   await prepareDirPath(modelDir, true);
   await prepareDirPath(interfacesPath, false);
-  const manifest = loadProjectManifest(projectPath);
+
+  const reader = await ReaderFactory.create(projectPath);
+  const manifest = parseProjectManifest(await reader.getProjectSchema());
```

Comment:
`reader.getProjectSchema()` seems always be used together with `parseProjectManifest()`, I think you can combine them together.